proto/pdpb.proto

```diff
@@ -261,6 +261,10 @@ message RegionHeartbeatResponse {
     ChangePeer change_peer = 2;
     // Pd can return transfer_leader to let TiKV does leader transfer itself.
     TransferLeader transfer_leader = 3;
+    uint64 region_id = 4;
+    metapb.RegionEpoch epoch = 5;
+    // Leader of the region at the moment of the corresponding request be made.
+    metapb.Peer original_leader = 6;
```

Comment:
actually, it may be not a leader peer if we support push later. So target peer may be more proper here.